Add v0.4.2

jni opened the pull request.

jni committed:
Update recipe to 0.4.2

jni committed:
MNT: Re-rendered with conda-smithy 2.4.5

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Using pinned numpy packages is a deprecated pattern.  Consider using the method outlined [here](https://conda-forge.org/docs/meta.html#building-against-numpy).

jni committed:
Change NumPy dependency pattern according to docs

See https://conda-forge.org/docs/meta.html#building-against-numpy

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.4.5

Now that the new `numpy` syntax has been adopted, re-render to update
the CI configurations accordingly.

jakirkham committed:
Fix typo in `numpy` version

Seems like `.x` should be ok, but only `.*` is allowed here. So fix this
accordingly.

jakirkham commented:
Pushed a few commits to this PR in the hopes of rectifying it. Appears there are some packaging issues though. Have submitted PR ( https://github.com/janelia-flyem/gala/pull/94 ), which should address them. Could switch to getting the source from GitHub here temporarily (should bypass these issues) or could do another patch release with that PR merged. Please let me know what you think.

jni commented:
I'll try to cut a new point release. I like keeping these two sources in sync.
